test/integrationTests/diagnostics.integration.test.ts

```diff
@@ -53,8 +53,8 @@ suite(`DiagnosticProvider: ${testAssetWorkspace.description}`, function () {
         this.skip(); // Remove this once https://github.com/OmniSharp/omnisharp-roslyn/issues/1458 is resolved.
```

Comment:
I tried to reactivate this test since the mentioned ticket is fixed, but it is failing on the `expect(ide0005).to.not.be.undefined`